src/main/java/com/revature/models/Wishlists.java

```diff
@@ -0,0 +1,23 @@
+package com.revature.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Wishlists {
```

Comment:
Naming convention. The class should be singular and we add `@Table(name = "wishlists")` to make sure our table name also follows DB convention of being plural.